opentyrian.appdata.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>opentyrian.desktop</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-2.0</project_license>
+  <name>OpenTyrian</name>
+  <summary>An open-source port of the DOS shoot-em-up Tyrian.</summary>
+  <description>
+    <p>OpenTyrian is an open-source port of the DOS game Tyrian.</p>
+    <p>Tyrian is an arcade-style vertical scrolling shooter.  The story is set in 20,031 where you play as Trent Hawkins, a skilled fighter-pilot employed to fight MicroSol and save the galaxy.</p>
+    <p>Tyrian features a story mode, one- and two-player arcade modes, and networked multiplayer.</p>
+  </description>
+  <url type="homepage">https://github.com/opentyrian/opentyrian</url>
+  <url type="bugtracker">https://github.com/opentyrian/opentyrian/issues</url>
+  <url type="faq">https://github.com/opentyrian/opentyrian/wiki/FAQ</url>
+  <content_rating type="oars-1.0">
+    <content_attribute id="violence-cartoon">mild</content_attribute>
+    <content_attribute id="violence-fantasy">none</content_attribute>
```

Comment:
Tyrian has the ["infamous ‘head on a spike’ picture that caused so much consternation."](https://lostgarden.home.blog/2007/04/05/free-game-graphics-tyrian-ships-and-tiles/)

Reply:
updated :)